fedimint-ui/apps/guardian-ui/src/GuardianApi.ts

```diff
@@ -79,9 +79,7 @@ export class GuardianApi implements ApiInterface {
         .catch((error) => {
           console.error('failed to open websocket', error);
           reject(
-            new Error(
-              'Failed to connect to API, confirm your server is online and try again.'
-            )
+            new Error('Failed to connect to API, confirm your server is online and try again.')
```

Comment:
We're using the Prettier formatter as defined in `fedimint-ui/package.json`, running `yarn format` from within `fedimint-ui` should format accordingly.